rocAL/include/file_io_stream.h

```diff
@@ -0,0 +1,22 @@
+
+#include <string>
```

Comment:
please remove this class and header file and use file c++ 17 file stream directly

Reply:
Done